src/sentry/discover/models.py

```diff
@@ -40,10 +41,18 @@ class DiscoverSavedQuery(Model):
     date_updated = models.DateTimeField(auto_now=True)
     visits = BoundedBigIntegerField(null=True, default=1)
     last_visited = models.DateTimeField(null=True, default=timezone.now)
+    is_homepage = models.BooleanField(null=True, blank=True)
```

Comment:
jfyi we can actually create columns with a default safely now, so you could create this as not null with default = False